internal/runners/deploy/path_lin_win.go

```diff
@@ -28,11 +29,14 @@ func usablePath() (string, error) {
 			continue
 		}
 
-		// Record result
+		// check for preferred PATHs
 		if funk.Contains(preferredPaths, path) {
 			return path, nil
 		}
-		result = path
+		// use the first available directory in PATH
+		if result == "" {
+			result = path
```

Comment:
This breaks the promise of the function; that it returns a "usable" path.

Reply:
I think it is still usable, no?

At this line, `path`  is a directory and writable ie., a path.  But now, I am only setting the *first* usable path to `result`.  Before, it was returning the *last* usable candidate, which created very strange results...  Usually, the first one on your PATH variable is the most important one...

Reply:
Oh I see, I miss-interpreted this.